test/installation/node/install from source without V8 snapshot for ARM

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+die () { echo $@ ; exit 1; }
+
+. ../../../nvm.sh
+
+NVM_TEST_VERSION=v0.10.7
+
+# Remove the stuff we're clobbering.
+[ -e ../../../$NVM_TEST_VERSION ] && rm -R ../../../$NVM_TEST_VERSION
+
+# Fake ARM arch
+nvm_get_arch() {
+  echo "armv7l"
+}
+
+# Install from source
+nvm install -s $NVM_TEST_VERSION || die "'nvm install -s $NVM_TEST_VERSION' failed"
+
+# Check Install
+[ -d ../../../$NVM_TEST_VERSION ]
+nvm run $NVM_TEST_VERSION --version | grep $NVM_TEST_VERSION || "'nvm run $NVM_TEST_VERSION --version | grep $NVM_TEST_VERSION' failed"
```

Comment:
After the install, the version should be auto-used, so you should be able to simply check `nvm current` and `node --version`, and below, directly use `node`